slices/xz-utils.yaml

```diff
@@ -0,0 +1,33 @@
+package: xz-utils
+
+essential:
+  - xz-utils_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - liblzma5_libs
+    contents:
+      /usr/bin/lzmainfo:
+      /usr/bin/unxz:
+      /usr/bin/xz:
+      /usr/bin/xzcat:
+  scripts:  # scripts which add xz compression to common utils
+    essential:
+      - base-files_tmp  # utils require temporary storage
+      # shebang in scripts reference interpreter in /bin not /usr/bin
+      - coreutils_bins
```

Comment:
just realized that coreutils brings a lot of stuff. do you need it all? the fact that it isn't a marked dependency makes it even more suspicious.

Reply:
`xzcmp` and `xzdiff` depend on coreutils for `mkdir` and `expr` unfortunately. Undeclared dependencies in apt was the core issue in the original PR. This is a good talking point in the upcoming sprint. :)

Also I found a missing change just above line 20. Should be fixed now

Reply:
If it's only a subset of coreutils bins that is needed, I actually think this is a perfect use case to explore the slicing power -> just create the slices for those binaries, thus ditching any other coreutils binaries that are not needed

> This is a good talking point in the upcoming sprint. :)

let's do it! (add a slot for it pls :) )

Reply:
Just pushed some changes for coreutils_mkdir and coreutils_expr slices + individual tests. The rootfs in the xzutils task should be much smaller now :)